vars/common_vars.yml.template

```diff
@@ -42,3 +42,5 @@ system_packages:
 create_swap_space: false
 
 emma_modules_dir: "/data/local/{{ lookup('env', 'CLUSTER_NAME') }}/modules"
+
+public_network_interface: "ansible_enp0s8"
```

Comment:
Why was this moved? It is still only used in the cassandra role.

Reply:
I was using it in Dask, but then I reverted it. Somehow I forgot to revert this one.
I just reverted the commit.